cmd/influx/pkg.go

```diff
@@ -91,6 +103,83 @@ func pkgApply(orgID, path *string, hasColor, hasTableBorders *bool) func(*cobra.
 	}
 }
 
+func pkgCreateCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "new",
+		Short: "Create a reusable pkg to create resources in a declarative manner",
+	}
+
+	wd, err := os.Getwd()
```

Comment:
to make it nicer, i will also add an output flag, so user to put it to anywhere they want

Reply:
totally agree, I added the following to handle that, the wd is purely for a default val.
https://github.com/influxdata/influxdb/pull/15937/files#diff-0ea570dc74aae6eb4768e34c0d584960R118

Reply:
looks like you already have a flag